walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -21,9 +21,28 @@
 import java.util.Iterator;
 
 public final class FindMeetingQuery {
+  /* If we can accomodate optional attendees, do so. Otherwise if there are required attendees,
+  ignore optional attendees.*/
+  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+    Collection<String> totalAttendees = new ArrayList<String>();
+    totalAttendees.addAll(request.getAttendees());
+    totalAttendees.addAll(request.getOptionalAttendees());
```

Comment:
Nit: the method name `getAttendees` makes me think that it gets all attendees. From the context in here I can tell that `getAttendees` only gives required attendees, but you might use it in other places in the future and forget to add context. Also having to add context every time you call a method is tedious.

You may want to rename it to something like `getRequiredAttendees`.

Additionally, variable names that begin with total are usually numbers. I'd suggest renaming `totalAttendees` to `allAttendees`.

Reply:
I will definitely change the totalAttendees variable! However, the getAttendees function was actually written for us (we did not create the meeting request class). I worry that changing its name may affect other pre-written dependencies.